sdk/core/azure-core/src/main/java/com/azure/core/util/logging/LogLevel.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.logging;
+
+import java.util.HashMap;
+import java.util.Locale;
+
+/**
+ * Enum which represent logging levels used in Azure SDKs. 
+ */
+public enum LogLevel {
```

Comment:
Loggers have distinct log levels for trace vs debug. Any reason we don't have equivalent of trace here?

Reply:
We have a discussion here, as there are different LogLevel for different logging framework, there is no need to have 1 on 1 corresponding. We are having minimum set here which is enough for current use. We can add more enums if there is need in future.